src/panoramic/cli/config/storage.py

```diff
@@ -16,7 +18,7 @@ def update_config(section: str, data: Dict[str, Any]) -> None:
 
 
 def read_config(section: str = '') -> Dict[str, Any]:
-    config_file = Paths.config_file()
+    config_file = Paths.context_file()
```

Comment:
since we're dropping the config file, can we rename everything from `config` to `context` ? Leaving it as is may create an unnecessary confusions for other contributors. what do you think?